packages/subgraph/tests/token/governance-erc20.test.ts

```diff
@@ -279,5 +278,61 @@ describe('Governance ERC20', () => {
       member.assertEntity();
       assert.entityCount('TokenVotingMember', 1);
     });
+
+    test('it should move delegation between members and end up deleting th member with no balances or delegated tokens', () => {
```

Comment:
Do we have any testing where we check the existence of an address as a member when balance > 0 and the absence of an address as a member when it drops to zero?

Reply:
this test does that, you have a member with voting power delegated to a member with no balance and a member with no delegation. The first undelegates so the second must dissapear and the third delegates to the first one so it ends up with no voting power but balance. At the end we check that the values of this 2 members are correct and that the second one was deleted by checking that the amount of members is 2

Reply:
mmm ok, maybe a test that checks just the simple scenario would be helpful, as well as this complex one. When the basics are clear, any complex things built on top should work as well

Reply:
```suggestion
    test('it should move delegation between members and end up deleting the member with no balances or delegated tokens', () => {
```